flowmachine/flowmachine/core/union.py

```diff
@@ -14,25 +14,26 @@ class Union(Query):
 
     Parameters
     ----------
-    top, bottom : flowmachine.Query object
-        First and second tables respectively
+    queries : flowmachine.Query object
+        Queries to union
+    all : bool, default: True
+        If True, queries will be deduped, if set to False, duplicate rows will be preserved.
     """
 
-    def __init__(self, top, bottom, all=True):
-        """"""
-
-        self.top = top
-        self.bottom = bottom
+    def __init__(self, *queries: Query, all: bool = True):
+        self.queries = queries
```

Comment:
Is it worth raising an error here if only one query is provided? The query would still work in that situation, but would be an entirely pointless wrapper around the input query.

Reply:
In fact, it wouldn't necessarily "work" for a single query, in that with `all=False` this should dedupe, which won't be the case for a single query.

Reply:
Worth an error on one query, yep.